spec/app_spec.rb

```diff
@@ -0,0 +1,111 @@
+require_relative "../app"
+require "colorize"
+
+def reset_tables
+  seed_sql = File.read('spec/orders_items_seeds.sql')
+  connection = PG.connect({ host: ENV['HOST'], dbname: 'shop_test', user: 'postgres', password: ENV['PASSWORD'] })
+  connection.exec(seed_sql)
+end
+
+RSpec.describe Application do
+  before(:each) do
+    reset_tables
+  end
+
+  let(:io) {double :io}
+  let(:app) {Application.new('shop_test', io, ItemRepository.new, OrderRepository.new)}
+
+  it "Loops until valid option selected and then exits" do
+
+    expect(io).to receive(:puts).with("\n")
```

Comment:
Although it's not super useful, I found it hard to read all the 'expect' lines together so created a mini-method to change them to 'gets' & 'puts'